doc/python/subplots.md

```diff
@@ -296,6 +296,24 @@ fig.update_layout(height=600, width=600,
 fig.show()
 ```
 
+### Subplots with Shared Colorscale
+
+To share colorscale information in multiple subplots, you can use [coloraxis](https://plot.ly/javascript/reference/#scatter-marker-line-coloraxis).
+
+```python
+from plotly.subplots import make_subplots
+import plotly.graph_objects as go
+fig = make_subplots(1, 2)
```

Comment:
here I would recommend adding `shared_yaxes=True` to make the bar heights comparable across the plots, then we can merge this!

Reply:
updated!